charts/enterprise/authelia/questions.yaml

```
@@ -433,34 +451,34 @@ questions:
                       description: "The groups filter used in search queries to find the groups of the user."
                       schema:
                         type: string
-                        default: ""
+                        default: "(member={dn})"
```

```suggestion
                        default: ""
```

User and ldap backend dependent.